application/views/interface_assets/footer.php

```diff
@@ -737,7 +737,7 @@ function showActivatorsMap(call, count, grids) {
     var maidenhead = new L.maidenheadactivators(grid_four).addTo(map);
 
     var osmUrl='<?php echo $this->optionslib->get_option('option_map_tile_server');?>';
-    var osmAttrib='Map data © <a href="https://openstreetmap.org">OpenStreetMap</a> contributors';
+    var osmAttrib='<?php echo $this->optionslib->get_option('option_map_tile_server_copyright');?>';
```

Comment:
@violarulan I guess this could be changed to the global variables as, instead of fetching from options, since we already set it on top.

Reply:
@AndreasK79 Thanks for the notice o7. I've created a new commit.